docker-compose.yml

```diff
@@ -18,6 +18,22 @@ services:
       - "18250:18250"
     volumes:
       - ./:/blockstore/app/:cached
+    depends_on:
+      - "datastore"
+
+  datastore:
+    container_name: datastore
+    image: mysql:8.0
```

Comment:
We'll be running on AWS RDS or Aurora, which only supports MySQL 5.x. So we shouldn't develop using MySQL 8.

Reply:
I'll update that accordingly. Thank you!